### PR TITLE
fix(learn): Consistent Styling in `Set up a Template Engine`

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

It was [mentioned in a previous PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/39653#issuecomment-697341137) that the `bold` tags in the text should actually be `code` tags. I've replaced them.

### CODE REVIEW

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ A template engine enables you to use static template files (such as those writte
 
 To set up <em>Pug</em> for use in your project, you will need to add it as a dependency in your package.json. Don't forget to add the name of the package and the version. Use the package.json for some examples of the correct syntax.
 
-Express needs to know which template engine you are using. We will use the <b>set</b> method to assign 'pug' as the <b>'view engine'</b>. <code>app.set('view engine', 'pug')</code>
+Express needs to know which template engine you are using. We will use the <code>set</code> method to assign <code>pug</code> as the <code>view engine</code>: <code>app.set('view engine', 'pug')</code>
 
-Your page will not load until you correctly <em>render</em> the index file in our <em>'views/pug'</em> directory.
+Your page will not load until you correctly <code>render</code> the index file in our <code>views/pug</code> directory.
```
Comment (Sky020) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md:20]: Whilst we are doing this, could you also change `our` to `the` please Nicholas? If that does not sound nice to you, then another option could be `your`

Comment (nhcarrigan) [reply curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md]: Taken care of 🙂. Thanks for the catch!

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ A template engine enables you to use static template files (such as those writte
 
 To set up <em>Pug</em> for use in your project, you will need to add it as a dependency in your package.json. Don't forget to add the name of the package and the version. Use the package.json for some examples of the correct syntax.
 
-Express needs to know which template engine you are using. We will use the <b>set</b> method to assign 'pug' as the <b>'view engine'</b>. <code>app.set('view engine', 'pug')</code>
+Express needs to know which template engine you are using. We will use the <code>set</code> method to assign <code>pug</code> as the <code>view engine</code>: <code>app.set('view engine', 'pug')</code>
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md:18]:
```suggestion
Express needs to know which template engine you are using. We will use the <code>set</code> method to assign <code>pug</code> as the <code>view engine</code> property's value: <code>app.set('view engine', 'pug')</code>
```

Comment (nhcarrigan) [reply curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md]: Sounds good! 🙂

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ A template engine enables you to use static template files (such as those writte
 
 To set up <em>Pug</em> for use in your project, you will need to add it as a dependency in your package.json. Don't forget to add the name of the package and the version. Use the package.json for some examples of the correct syntax.
 
-Express needs to know which template engine you are using. We will use the <b>set</b> method to assign 'pug' as the <b>'view engine'</b>. <code>app.set('view engine', 'pug')</code>
+Express needs to know which template engine you are using. We will use the <code>set</code> method to assign <code>pug</code> as the <code>view engine</code> property's value: <code>app.set('view engine', 'pug')</code>
 
-Your page will not load until you correctly <em>render</em> the index file in our <em>'views/pug'</em> directory.
+Your page will not load until you correctly <code>render</code> the index file in the <code>views/pug</code> directory.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-a-template-engine.english.md:20]:
```suggestion
Your page will not load until you correctly <code>render</code> the index file in the <code>views/pug</code> directory.
```
"render" is just a verb here, so I think it does not need code tags.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
LGTM

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 

Randell, if you are still happy with this, by all means merge. I will add: I think we will circle back around to this file (amongst other files) in another PR, one day.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**